submissions/omawumiRebeccaMoju/html-forms/contact form.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>BLK_My Naturals</title>
+</head>
+
+<body>
+    <h1>BLK_My Naturals</h1>
+    <form>
+        <fieldset>
+            <legend>Contact Form</legend>
+            
+            <label for="firstname">First Name</label><br>
+            <input type="text" name="First name" id="firstname"/>
+            <br />
```

Comment:
Remove the space before / in the closing tag. Check other places and make changes